src/javascript/app_2/Modules/Trading/Containers/trade.jsx

```diff
@@ -30,11 +33,20 @@ class Trade extends React.Component {
                     }
                     <Test />
                 </div>
-                { contract_id ?
-                    <ContractDetails contract_id={contract_id} onClickNewTrade={this.props.onClickNewTrade} />
-                    :
-                    <FormLayout is_mobile={this.props.is_mobile} is_trade_enabled={this.props.is_trade_enabled} />
-                }
+                <div className={classNames(form_wrapper_class, slider_class)}>
+                    {
+                        contract_id &&
+                            <ContractDetails
+                                contract_id={contract_id}
+                                onClickNewTrade={this.props.onClickNewTrade}
+                            />
+                    }
+                    <FormLayout
+                        is_mobile={this.props.is_mobile}
+                        is_contract_visible={contract_id}
```

Comment:
According to the property name and its purpose, better to assign `!!contract_id` value and use boolean type everywhere.